study.md

```diff
@@ -43,5 +48,8 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+Hash URLs are URLs that contain a hashtag (#), which loads the starting state of the application and allows you to move around the site while keeping the app state in sync. Example:  #/posts/new  takes you to posts.new route.
```

Comment:
Hash URLs will drop the # when they are copied and pasted, 
> Example: #/posts/new  takes you to posts.new route.
so with somesite.com/#/post/new, it would be somesite.com/post/new, which would loose the view that was being shared.